utilities_common/db.py

```diff
@@ -0,0 +1,6 @@
+from swsssdk import ConfigDBConnector
+
+class Db(object):
+    def __init__(self):
+        self.cdb = ConfigDBConnector()
+        self.cdb.connect()
```

Comment:
later we will need to expand this object to support various DB, state db, and also multi-asic support.

Reply:
Should we rename `cdb` to be more descriptive, like `config_db` or `configdb`

Reply:
i am mostly follow the convention in swss vs tests. 

https://github.com/Azure/sonic-swss/blob/82d36c4f7ded1eceb832ab737fc13e6ad145090c/tests/test_port_mac_learn.py#L8

i mostly prefer abbrev as the code is shorter.

Reply:
I see. Shorter is convenient, but it can get confusing, especially as more DBs are added . I see there is now `cntdb` in the swss vs tests. Maybe we should consider transitioning to 3/4-letter abbreviations (`appdb`, cfgdb`, `cntdb`, `asicdb`, etc.)?

Reply:
sure. changed to cfgdb